cylc/flow/task_trigger.py

```diff
@@ -106,7 +96,8 @@ def get_trigger_name(trigger_name):
         for standard_name, alt_names in _ALT_TRIGGER_NAMES.items():
             if trigger_name == standard_name or trigger_name in alt_names:
                 return standard_name
-        raise TaskTriggerError("Illegal task trigger name: %s" % trigger_name)
+        raise TriggerExpressionError(
+            "Illegal task trigger name: %s" % trigger_name)
```

Comment:
If you update the PR, maybe f-string this one?

Reply:
https://www.python.org/dev/peps/pep-0498/